.travis.yml

```diff
@@ -21,10 +20,14 @@ env:
 
 matrix:
   fast_finish: true
+  include:
+    - php: 5.6
+      env: SYMFONY_VERSION="2.8.*"
```

Comment:
I would include here as well "^3.0.0". At this moment this is not working, but we could add it in allow failures.
Is not working because of GaufretteBundle.